cmd/monoctl/get/get.go

```diff
@@ -50,11 +55,14 @@ func NewGetCmd() *cobra.Command {
 	cmd.AddCommand(NewGetTenantUsersCmd())
 	cmd.AddCommand(NewGetClusterCredentials())
 	cmd.AddCommand(NewGetClusterAccess())
+	cmd.AddCommand(NewGetAuditLogCmd())
 
 	flags := cmd.PersistentFlags()
 	flags.BoolVarP(&showDeleted, "deleted", "d", false, "Show deleted resources.")
 	flags.StringVar(&sortBy, "sort-by", "", "Column to sort result by. Uses the first column by default.")
 	flags.BoolVar(&sortDescending, "descending", false, "Sort result in descending order.")
+	flags.StringVar(&exportFile, "export", "", "exports the output to a file in CSV format. If no file is specified m8-output.csv will be written in the current directory if it doesn't exists")
```

Comment:
You can set the default in the third argument instead of an extra function call one line later.

Reply:
Yes and no. I couldn't figure out how to nicely get a boolean from a string flag to know if the user wants to export (to the default value) so I resorted to checking if `exportFile` is empty. Putting this in a one-liner will mean that `exportFile` has a value all the time which make it indistinguishable between the user using --export only or --export custom/path/to/file.csv. `NoOptDefVal` will assign `exportFile` the default value only if the user used --export without an argument

Reply:
allright, thanks for the explanation